### PR TITLE
Photogram industrial 2

### PR DESCRIPTION
**Pull request by joannarodriguez134**

### CODE REVIEW

**Review by flealc — APPROVED** [review]
Looks good! Merging is optional at this point. I would wait and merge everything at the end, once you have completed all parts of the assignment.

**Comment by Shafaq-Juda** [comment]
dsd

File: Gemfile
Comment (lsampson1974) [inline Gemfile:1]: This is good.

File: db/migrate/20240227213312_create_comments.rb
Comment (lsampson1974) [inline db/migrate/20240227213312_create_comments.rb:1]: This is good.

File: db/schema.rb
Comment (lsampson1974) [inline db/schema.rb:1]: This is weird !!

**Review by lsampson1974 — COMMENTED** [review]
LTGM

**Pull request merged**